packages/mobile/locales/base/translation.json

```diff
@@ -266,6 +266,7 @@
   "withdrawScanQrTitle": "Scan",
   "tokenBalance": "{{token}} Balance",
   "addFunds": "Add Funds",
```

Comment:
Is this string used anymore? can it be deleted?

Reply:
yeah, it's used in other places such as FiatExchange.tsx and FiatExchangeOptions.tsx